apps/web/src/app/(pages)/(dashboard)/redirection/page.tsx

```diff
@@ -0,0 +1,15 @@
+export default function ErrorPage() {
+  return (
+    <div className="flex h-[85vh] flex-col">
```

Comment:
This is jobless behavior of me but the header is 20dvh so if we make this 80dvh then it would be the full screen, and then scrolling at all would reveal the footer. So would we be able to change this  🙈